src/executables/util_cs.c

```diff
@@ -3202,25 +3203,30 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   check_applied_info = utility_get_option_bool_value (arg_map, APPLYINFO_APPLIED_INFO_S);
   log_path = utility_get_option_string_value (arg_map, APPLYINFO_COPIED_LOG_PATH_S, 0);
+
   if (log_path != NULL)
     {
-      log_path = realpath (log_path, log_path_buf);
+      check_copied_info = true;
+      if (realpath (log_path, log_path_buf) != NULL)
```

Comment:
I remembered that `if (realpath (log_path, log_path_buf) != NULL)` will disturb the readability.

But it is essential code for query object to be loaded.

Reply:
I remembered that I told you how about calling 'realpath' in the later code in which you handle the log path.

Reply:
Right. But `realpath()` should be used before check '-a' option. it removes the slash(/) of `log_path`. and then it is able to get query results.

Is this position of code no problem?